test_scripts/Defects/7_1/3580_Revoked_app_activated_via_device_consent.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/3580
+--
+-- Description:
+-- HMI responds with UNSUPPORTED_RESOURCE to Speak component of SubtleAlert
```

Comment:
Update description

Reply:
Added in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2505/commits/7ffb9e92e9d995f843068e807a2f61437c5db45b